bpf/tc/compiler.go

```diff
@@ -48,6 +48,20 @@ const (
 	EpTypeWireguard EndpointType = "wireguard"
 )
 
+type ProgName string
+
+const (
+	policyProgram ProgName = "calico_tc_norm_pol_tail"
+	allowProgram  ProgName = "calico_tc_skb_accepted_entrypoint"
+	icmpProgram   ProgName = "calico_tc_skb_send_icmp_replies"
+)
+
+const (
+	POLICY_PROGRAM_INDEX = iota
```

Comment:
Nit: golang naming convention is to name constants like normal variables. PolicyProgramIndex, for example.